instrumentserver/apps.py

```diff
@@ -58,25 +59,34 @@ def serverScript() -> None:
     # Load and process the config file if any.
     configPath = args.config
 
-    stationConfig, serverConfig, guiConfig, tempFile = None, None, None, None
+    stationConfig, serverConfig, guiConfig, tempFile, pollingRates = None, None, None, None, None
     if configPath != '':
-        # Separates the corresponding settings into the 4 necessary parts
-        stationConfig, serverConfig, guiConfig, tempFile = loadConfig(configPath)
+        # Separates the corresponding settings into the 5 necessary parts
+        stationConfig, serverConfig, guiConfig, tempFile, pollingRates = loadConfig(configPath)
+
+    pollingThread = QtCore.QThread()
```

Comment:
Add an if that thins only happens if pollinRates is not None, no need to start a new polling thread if no polling is accepted.